openedx/adg/lms/applications/tests/test_helpers.py

```diff
@@ -352,3 +319,18 @@ def test_has_admin_permissions(mocker, is_business_line_admin, is_superuser, is_
     test_user = UserFactory(is_superuser=is_superuser, is_staff=is_staff)
 
     assert has_admin_permissions(test_user) == expected_result
+
+
+@pytest.mark.django_db
+def test_validate_word_limit_text_area():
+    """
+    Check if the `validate_word_limit_text_area` function raises a ValidationError if the total
+    number of words exceed the provided limit
+    """
+    text_area_input = 'test word limit for textarea'
+    assert validate_word_limit_text_area(text_area_input) is None
```

Comment:
`is None` is not required (no pun intended)

Reply:
I actually wanted it to be falsy.
So, either `assert not validate_word_limit_text_area` or `assert validate_word_limit_text_area is None` is needed. 

I'll go with the former.